functions/go/apply-setters/applysetters/apply_setters_test.go

```diff
@@ -315,6 +315,56 @@ spec:
 `,
 			errMsg: `invalid setter pattern for array node: "${images}:${tag}"`,
 		},
+		{
+			name: "scalar partial setter using dots",
+			config: `
+data:
+  domain: demo
+  tld: io
+`,
+			input: `apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  annotations:
+    description: The route for the my-app-layer http service.
+  labels:
+    app: my-app
+    component: my-app-layer
+    stage: dev # kpt-set: ${stage}
+    version: 0.2.6 # kpt-set: ${version}
+  name: my-app-layer
+spec:
+  host: my-app-layer.dev.example.com # kpt-set: my-app-layer.${stage}.${domain}.${tld}
+  port:
```

Comment:
Remove section after this for yaml to be concise.

Reply:
I've changed it to a config map and removed all the verbosity.